.circleci/config.yml

```diff
@@ -166,18 +166,27 @@ jobs:
         name: Remove last snapshot and backup database
         no_output_timeout: 15m
         command: |
-          DESCRIBE_SNAPSHOT=`aws rds describe-db-snapshots --db-snapshot-identifier tm3-<< parameters.stack_name >>-$RDS_ID-latest --db-instance-identifier $RDS_ID --output text`
+          DESCRIBE_SNAPSHOT=$(aws rds describe-db-snapshots --db-snapshot-identifier tm3-<< parameters.stack_name >>-$RDS_ID-latest --db-instance-identifier $RDS_ID --output text)
+          NONCE=$(openssl rand -hex 4)
+          echo "export NONCE=$NONCE" >> $BASH_ENV
           # Copy old snapshot to temporary
           if [ -z "$DESCRIBE_SNAPSHOT" ]
           then
               echo "Snapshot does not exist, creating one now."
           else
-              aws rds copy-db-snapshot --source-db-snapshot tm3-<< parameters.stack_name >>-$RDS_ID-latest --target-db-snapshot tm3-<< parameters.stack_name >>-$RDS_ID-temp
-              aws rds delete-db-snapshot --db-snapshot-identifier tm3-<< parameters.stack_name >>-$RDS_ID-latest
+              aws rds copy-db-snapshot \
+                      --source-db-snapshot tm3-<< parameters.stack_name >>-$RDS_ID-latest \
```

Comment:
Should `$RDS_ID` be in `{}` here for consistency?

Reply:
All bash variables throughout should ideally be within `{}` for consistency. I'll change it.

Reply:
Done.